docs/next.config.js

```diff
@@ -179,7 +179,10 @@ module.exports = {
     if (process.env.PULL_REQUEST === 'true' && !l10nPRInNetlify) {
       // eslint-disable-next-line no-console
       console.log('Considering only English for SSR');
-      traverse(pages, 'en');
+      // traverse(pages, 'en');
+      LANGUAGES_SSR.forEach((userLanguage) => {
+        traverse(pages, userLanguage);
+      });
```

Comment:
To apply before we merge

```suggestion
      traverse(pages, 'en');
```